test/challtestsrv/dnsone.go

```diff
@@ -101,6 +102,9 @@ func (s *ChallSrv) dnsHandler(w dns.ResponseWriter, r *dns.Msg) {
 				value = "sad-hacker-ca.invalid"
 			case "good-caa-reserved.com.":
 				value = "happy-hacker-ca.invalid"
+			case "account-uri.good-caa-reserved.com.":
+				uri := os.Getenv("ACCOUNT_URI")
```

Comment:
Can you set a default `ACCOUNT_URI` here? I think it will make the failure clearer than if `ACCOUNT_URI` were unset and the record ended up being `happy-hacker-ca.invalid; account=uri=`

Reply:
It seems that seeing `account-uri=` is easier to realise that something went wrong with test setup, instead of seeing `account-uri=http://boulder:4000/acct/reg/12345` and wondering why on earth the account values do not match. If we want to use something other than an empty string, I'd opt for "unspecified" or some value that makes it obvious that this is a test data setup failure.

Let me know if you still want this changed.

Reply:
@4a6f656c That's a fair argument. I think you're right. I can imagine eventually we may have a way to set response records in the challtestsrv through some kind of external API instead of hardcoding them and if I were implementing that I wouldn't think of using a default value for a account-uri/CAA record if none were provided. 

Let's leave this as is. Thanks for entertaining the feedback :+1: